### ISSUE TITLE
Where's the pom?

### ISSUE THREAD

**Issue by RoboMWM** [Maxetto/AntiJoinBot#1]
Is this mavenized? The directory structure suggests it would be but there's no pom...

**Comment by Maxetto** [comment]
No, this is not a Maven project (Yet?). Don't even know if I'll Mavenize it. I don't feel it's necessary but who knows...

**Comment by Oruss7** [comment]
to building server dependencies with maven, that might be cool =)

**Comment by sgdc3** [comment]
@Maxetto is too lazy, i'll send a PR if he wants ;)

**Comment by Maxetto** [comment]
@sgdc3 I'm not lazy, I just don't know how to use Maven :P
I don't even think it should be useful, we are not hooking with many projects and the ones we hook are already shaded.

Version 6 might be different

**Comment by RoboMWM** [comment]
Let's just say it'd definitely help contributing since I wouldn't need to setup anything. Plus a build system is nice.

That being said it's really easy to use maven, but I'm just waiting for this "recode" and/or might just develop something of my own, idk.

**Comment by Maxetto** [comment]
If you'd like to contribute, feel free to do it.
You just need to download the source and import it to Eclipse. You only need Spigot as an external library and you're ready to go.

The recode is something that wont happen soon since i'm near my final exams. I wont probably work at it before July.
